.github/workflows/ci.yaml

```diff
@@ -187,12 +187,6 @@ jobs:
         with:
           key: ubuntu-latest
           cache-on-failure: true
-      - uses: actions/setup-go@v3             # Install Go for http mirroring tests with a Go webserver.
```

Comment:
We're gonna re-add these tests later when they're fixed right? So isn't it better to just comment these lines instead of removing them (and add a TODO)?

Reply:
Changed deletions to comments.